float32 angle
int32 class_id
int32 octave
string pt
float32 response
float32 size
